spec/models/translation_override_spec.rb

```diff
@@ -283,4 +283,43 @@ def cached_value(guardian, translation_key, locale:)
       end
     end
   end
+
+  describe "#original_translation_updated?" do
+    context "when the translation is up to date" do
+      fab!(:translation) { Fabricate(:translation_override, translation_key: "title") }
+
+      it { expect(translation.original_translation_updated?).to eq(false) }
+    end
+
+    context "when the translation is outdated" do
+      fab!(:translation) do
+        Fabricate(:translation_override, translation_key: "title", original_translation: "outdated")
+      end
+
+      it { expect(translation.original_translation_updated?).to eq(true) }
+    end
+
+    context "when we can't tell because the translation is too old" do
+      fab!(:translation) do
+        Fabricate(:translation_override, translation_key: "title", original_translation: nil)
+      end
+
+      it { expect(translation.original_translation_updated?).to eq(false) }
+    end
+  end
+
+  describe "invalid_interpolation_keys" do
+    fab!(:translation) do
```

Comment:
I think we'll need to fabricate valid translation overrides to ensure we're actually filtering those out.